components/rmrk/ExploreLayout/ExploreLayout.vue

```diff
@@ -4,61 +4,77 @@
       {{ $t('general.searchResultsText') }}
       <span class="text__stroked is-size-3">{{ $route.query.search }}</span>
     </div>
-    <b-tabs v-model="selectedTab" data-cy="tabs" :animated="false">
-      <b-tab-item :label="$t('collections')" value="COLLECTION">
-        <template v-if="selectedTab === 'COLLECTION'">
+    <o-tabs
+      v-model="selectedTab"
+      data-cy="tabs"
+      class="explore-tabs"
+      content-class="explore-tabs-content">
+      <o-tab-item :label="$t('collections')" value="collectibles">
```

Comment:
![thats-bad](https://user-images.githubusercontent.com/9987732/210205438-878098b2-9efe-4a28-8e0f-7a6ee6cf863d.gif)

Reply:
@roiLeo should we not use <o-tabs here?

Reply:
oruga-ui is integrated in `libs/ui`, the day we take this dependency outside the project (like if we need this lib on another project) it will break.

Reply:
@roiLeo but it seems we already use it here 
<img width="484" alt="Screen Shot 2023-01-02 at 1 55 00 PM" src="https://user-images.githubusercontent.com/39299315/210280415-8eacc004-2ef8-4d72-9b0e-5cc54b6f2397.png">
and 
<img width="590" alt="Screen Shot 2023-01-02 at 1 55 16 PM" src="https://user-images.githubusercontent.com/39299315/210280425-90cd28c3-c01d-4a6e-9bd4-083f5cb3ddbf.png">
These are not in libs/ui , is this wrong too then?

Reply:
`DisablableTab` is already in libs/ui
we should wrap `o-tabs` component in a new histoire component, this shouldn't have been done like this at the start